docker_scripts/config-docker-sat.sh

```diff
@@ -27,10 +27,10 @@ LOGDIR=/var/log/cray/sat
 
 SATMANDIR=/usr/share/man/man8
 
-CSM_RPMS_REPO="https://github.com/Cray-HPE/csm-rpms.git"
-CSM_RPMS_DIR="csm-rpms"
-CSM_RPMS_BASE_PACKAGES_PATH="packages/node-image-ncn-common/base.packages"
-CSM_RPMS_BRANCH="main"
+METAL_PROVISION_REPO="https://github.com/Cray-HPE/metal-provision.git"
+METAL_PROVISION_DIR="metal-provision"
+METAL_PROVISION_BASE_PACKAGES_PATH="roles/node-images-ncn-common/vars/packages/suse.yml"
+METAL_PROVISION_BRANCH="main"
 KUBERNETES_VERSION_REGEX="[0-9]+\.[0-9]+\.[0-9]+"
```

Comment:
This variable is no longer used and could be removed to avoid confusion.

Reply:
Ah, good catch.